src/main/java/co/omise/models/SearchResult.java

```diff
@@ -124,10 +137,19 @@ public Options order(Ordering order) {
                     builder = builder.put("filters[" + entry.getKey() + "]", entry.getValue());
                 }
             }
+
             if (order != null) {
                 builder = builder.put("order", serializer.serializeToQueryParams(order));
             }
 
+            if (page > 0) {
+                builder = builder.put("page", page + "");
```

Comment:
Could be better to use `String.valueOf(int)`

Reply:
👌